core/App/ehsm_napi.cpp

```diff
@@ -80,6 +80,22 @@ typedef struct {
         retString += "}";
 		return StringToChar(retString);
 	};
+
+    char* parseStringData(char* jsonChar, std::string key) const{
```

Comment:
the function name not very fitable, for the implementation , it seems just for using to retrieve the "key" value, right?


and who will call it?

Reply:
yes. It simply parses string data, which will be used in unit tests.
You can see it in ehsm_core_test.cpp of the next PR.

Reply:
just see that in the json struct, there's already has a function named as "void addData(string key, string data)", so why not add another function as "void readData(string key, string data)"

Also, you need to re-submit the PR together with the changes in the unittest code.